### PR TITLE
Change section name of WND to distinguish it from the server one

### PR DESCRIPTION
**Pull request by voroyam**
often if searching for the server wnd guide you get to the appliance guide and there is no simple way to distinguish what is what in the search.

the name change should fix this.

### CODE REVIEW

File: modules/admin_manual/pages/appliance/configuration/wnd_setup.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= Windows Network Drive Configuration (WND)
+= WND in the Appliance (WND)
```
Comment (phil-davis) [inline modules/admin_manual/pages/appliance/configuration/wnd_setup.adoc:1]:
```suggestion
= Windows Network Drive in the Appliance (WND)
```

The acronym is already mentioned at the end of the title. If you use the acronym at the start also, then the meaning of "WND" does not appear in the document.

Comment (voroyam) [reply modules/admin_manual/pages/appliance/configuration/wnd_setup.adoc:1]: My worry is the character limitation in the search window. I want the title to be very distinct from the WND one and from a future document WND Quick Guide.

So we have Configuring Windows Network Drives, WND in the Appliance and WND quick guide.

Comment (enbrnz) [reply modules/admin_manual/pages/appliance/configuration/wnd_setup.adoc:1]: I'm confused now. Either write it out and add the abbreviation at the end, or just abbreviate in the front and neglect the abbreviation at the end. But don't abbreviate in the front and at the end in parenthesis, that's redundant.

If there is some character limit in the search, I vote for these three article headlines:
- Windows Network Drive (WND)
- WND Quick Guide
- WND in the Appliance

**Review by enbrnz — APPROVED** [review]

File: modules/admin_manual/pages/appliance/configuration/wnd_setup.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/appliance/configuration/wnd_setup.adoc]: valid point. I have forgotten to remove the abbreviation at the end.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
I will make 2 backports to 10.6 and 10.5...

**Comment by phil-davis** [comment]
bACKPORTS:
10.5 #3133 
10.6 #3134

**Pull request merged**